api/config/packages/security.yml

```diff
@@ -91,7 +91,8 @@ security:
 
     # https://opgtransform.atlassian.net/wiki/spaces/DEPDS/pages/191299647/User+roles
     role_hierarchy:
-        ROLE_SUPER_ADMIN:     [ ROLE_ADMIN, ROLE_ELEVATED_ADMIN ]
+        ROLE_SUPER_ADMIN:     [ ROLE_ADMIN, ROLE_ADMIN_MANAGER ]
```

Comment:
Question.. as neither role_admin nor role_admin_manager can create admin manager roles, is super admin still getting those permissions from somewhere?

Reply:
Cool :)